docs/configuration.asciidoc

```diff
@@ -1,5 +1,5 @@
 ////
-This file is auto generated
+This file is auto-generated 
 
 Please only make changes in configuration.asciidoc.ftl
 ////
```

Comment:
On line 9, should agent here be in {} or no? It's hard to know what should be and should be when it's being referenced.

Reply:
I get that. Any GitHub repo prefixed with `apm-` should not use `{agent}`.